wagtail/admin/templates/wagtailadmin/panels/tabbed_interface.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Pattern, generateDocs } from 'storybook-django/src/react';
+import { initTabs } from '../../../../../client/src/includes/tabs';
+import template from './tabbed_interface.html';
+
+document.addEventListener('DOMContentLoaded', () => {
+  initTabs();
+});
+
+const { docs, argTypes } = generateDocs(template);
+
+export default {
+  title: 'Shared / Tabs / TabbedInterface',
+  parameters: { docs },
+  argTypes: { ...argTypes },
+};
+
+const Template = (args) => <Pattern filename={__filename} context={args} />;
+
+export const Base = Template.bind({});
+
+Base.args = {
+  self: {
+    visible_children: [
+      {
+        heading: 'Content',
+        render_as_object: `<h1>Content</h1><div>Content Body</div>`,
+      },
+      {
+        heading: 'Promote',
+        render_as_object: `<h1>Promote</h1><div>Promote Body</div>`,
+      },
+      {
+        heading: 'Settings',
+        render_as_object: `<h1>Setting</h1><div>Settings Body</div>`,
+      },
+    ],
+  },
+};
+export const Single = Template.bind({});
+Single.args = {
+  self: {
+    visible_children: [
+      {
+        heading: 'Tab1',
+        render_as_object: `<h1>Title1</h1><div>Body Text</div>`,
+      },
+    ],
+  },
+};
+export const Multiple = Template.bind({});
```

Comment:
minor nitpick but please can there be a line break after each class and each export in these files - just makes it a bit more readable.

```suggestion

export const Multiple = Template.bind({});

```